src/main/java/reposense/commits/CommitResultAggregator.java

```diff
@@ -104,6 +112,14 @@ private static float getContributionVariance(List<AuthorDailyContribution> contr
         return variance / totalDays;
     }
 
+    /**
+     * Creates a {@link Map} of each author in {@code authorSet} to their respective {@code commitResults}
```

Comment:
I think the first sentence here can be `Returns a mapping of each {@link author} to their commit contributions.` The rest of it seems to go quite into detail on how they are represented which can be placed in the second sentence and onwards.